unfoldingWord/translationCore
# added command to generate osx releases

#### This pull request addresses:

This is just a small update that adds a command in package.json to generate osx releases.
This is loosely related to https://github.com/unfoldingWord-dev/translationCore/issues/2807.

#### How to test this pull request:

You should be able to run `npm run release-osx` on a linux box or Mac to build a dmg. On a Mac this will create a compressed dmg.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3441)
